python/src/worker/tasks/differential_expression.py

```diff
@@ -130,9 +138,12 @@ def compute(self):
         result = pandas.DataFrame.from_dict(r.json())
 
         result.dropna(inplace=True)
-
+        total = 0
+        if len(result) > 0:
```

Comment:
You don't need this if `result["full_count"]` returns an integer

Reply:
Replace for {df: data.frame, total: totalCount}